src/layers/conv.jl

```diff
@@ -1,168 +1,99 @@
 """
-    conv_bn(kernelsize, inplanes, outplanes, activation = relu;
-                 rev = false, preact = false, use_bn = true, stride = 1, pad = 0, dilation = 1,
-                 groups = 1, [bias, weight, init], initβ = Flux.zeros32, initγ = Flux.ones32,
-                 ϵ = 1.0f-5, momentum = 1.0f-1)
+    conv_norm(kernel_size, inplanes::Int, outplanes::Int, activation = relu;
+              norm_layer = BatchNorm, prenorm = false, preact = false, use_bn = true,
+              stride = 1, pad = 0, dilation = 1, groups = 1, [bias, weight, init])
 
 Create a convolution + batch normalization pair with activation.
 
 # Arguments
 
-  - `kernelsize`: size of the convolution kernel (tuple)
+  - `kernel_size`: size of the convolution kernel (tuple)
   - `inplanes`: number of input feature maps
   - `outplanes`: number of output feature maps
   - `activation`: the activation function for the final layer
-  - `rev`: set to `true` to place the batch norm before the convolution
+  - `norm_layer`: the normalization layer used
+  - `prenorm`: set to `true` to place the batch norm before the convolution
```

Comment:
I would say `revnorm` makes more sense to me over `prenorm`.

Reply:
`prenorm` evokes memories of the similar concept in ViTs, hence the nomenclature. Can always flip

Reply:
Done